payloads/library/recon/simple-tcpdump/README.md

```diff
@@ -0,0 +1,7 @@
+- Install following packages : ``` coreutils-timeout zip ```
+- Refer to this payload to install package https://github.com/julesbozouklian/shark_jack_payload/blob/main/payload/util/install_package.sh
```

Comment:
Any chance you could put this into this PR instead? Would be easier to update in the future.

Reply:
yes i update my PR

Reply:
ok it's updated